extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -191,12 +193,24 @@ function BaseCarouselWithRef(
     return interaction.current === Interaction.TOUCH;
   }, [controls, outsetArrows]);
 
+  const [rtl, setRtl] = useState(false);
```

Comment:
Should we also make this overridable with a direct component prop? E.g. `dir: ltr|rtl|auto`?

Reply:
Sure, added.